__tests__/components/__snapshots__/accountVerification.test.js.snap

```diff
@@ -0,0 +1,140 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
remove comments please

Reply:
This one is auto generated by jest. It is generated again whenever I run tests again